model/shared/src/main/scala/explore/model/TargetVisualOptions.scala

```diff
@@ -10,31 +10,31 @@ import lucuma.core.math.syntax.int._
 import monocle.Focus
 
 final case class TargetVisualOptions(
-  fov:      Visible,
-  fovAngle: Angle,
+  ccd:      Visible,
+  fov:      Option[Angle],
```

Comment:
I think this could better be a `Pot[Angle]` however I start having problems with the combinations of `ReuseView` and `Pot` especially if I need to zoom on the options to get the the angle